src/ui/components/SecondaryToolbox/ReduxDevTools.tsx

```diff
@@ -68,9 +68,10 @@ function ActionItem({
   selectedPoint: ExecutionPoint | null;
   setSelectedPoint: (point: ExecutionPoint | null) => void;
 }) {
+  const sourcesState = useAppSelector(state => state.sources);
```

Comment:
:hand: This should be moved into the thunk as a `getState()` - the UI doesn't need this part